MREUnityRuntime/MREUnityRuntimeLib/Util/Unity/AssetFetcher.cs

```diff
@@ -8,9 +8,9 @@ namespace MixedRealityExtension.Util.Unity
 
     public static class AssetFetcher<T> where T : class
     {
-        public struct FetchResult<T>
+        public struct FetchResult<TT> where TT : T
```

Comment:
>TT [](start = 34, length = 2)

Did you mean to check this in? This seems unrelated to LookAt.

Reply:
Fixing a build warning: T shadowed T from parent class.